### DISCUSSION TITLE
New Home page gives no option to display commits in repo's I watch

### DISCUSSION THREAD

**Discussion by hostep** [community/community#66200] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The old home page was great for me, I follow a lot of repo's that we use for work, to see what's going on, what direction they are moving in, what bugs they fix and so on and so on.

However, the new home page no longer shows me some of those parts.
One big thing that lacks for me, is to be able to see all commits and merges (ones without a pull request) that are added to a repo. This is something that was very valuable to me and my professional work.
It would be great if this feature can be brought back to the homepage.

Thanks for listening!

**Comment by Ali-RS** [comment]
Agree with you, it's very unfortunate that they have removed the recent commits on the new feed/home page which was very valuable. Not sure what was the reason behind this decision, hope they bring it back.

**Comment by RumeysaaKopuz** [comment]
Absolutely I agree #1

**Comment by Ali-RS** [comment]
It seems for now we can use:  https://github.com/dashboard-feed

**Comment by andrewjswan** [comment]
Agree with you, it's very unfortunate that they have removed the recent commits on the new feed/home page which was very valuable.

**Comment by mrwunderbar666** [comment]
I also agree, I want to see recent commits to repositories that I watch. I wonder why GitHub does not enable that option or at least add a new screen / tab where I can quickly see that

**Comment by hrach** [comment]
I need to follow the new code for AdventOfCode being pushed every day... impossible :(

**Comment by joangava** [comment]
ola boa noite eu sou novo por aqui ; estou a precisar da ajuda vossa para o meu trabalho . nao conheço nada disso por isso . estou ca para vous perdir o vosso concelho como posso me proceder para que descubro e recoltar todos os meus dados . agradeço inperativamente a vossa ajuda . 
muito obrigado a cada um de voces .

**Comment by mohamedayoub97** [comment]
I totally understand your frustration. The ability to track all commits and merges directly from the homepage was a key feature for many of us. While the new homepage focuses on different aspects, it would definitely be great if there were an option to bring back this functionality or at least offer it as a setting.

In the meantime, you can use the **_[dashboard feed]_** as a workaround to see recent activity. Hopefully, the team will consider reintroducing this feature soon based on user feedback.

Thanks for sharing your thoughts!
